src/frontend/src/lib/components/send/SendInputAmount.svelte

```diff
@@ -22,6 +22,8 @@
 		amount = calculateMax?.();
 	};
 
+	// This function is used for test scopes to forcibly trigger the onMax function.
```

Comment:
The function `triggerCalculateMax` is used in `SendAmount` component.

Reply:
will remove the comment